src/Components/HighChartsWrapper/HighChartsWrapper.types.ts

```diff
@@ -109,3 +110,11 @@ export const HighChartsMockData: Array<IHighChartSeriesData> = [
         tooltipSuffix: 'm'
     }
 ];
+
+export const HighChartsMockLargeData: Array<IHighChartSeriesData> = [
```

Comment:
can we please pull out mock data to a .mock file so it's not in our production code base?